media-gfx/xpaint/xpaint-2.10.2-r1.ebuild

```diff
@@ -36,7 +36,9 @@ RDEPEND="
 	)
 "
 DEPEND="${RDEPEND}"
+# sys-devel/automake:1.15 for bug #778968; not needed in xpaint 3+
 BDEPEND="
+	sys-devel/automake:1.15
```

Comment:
it is a wrong way to depend on automake. Please consult https://devmanual.gentoo.org/eclass-reference/autotools.eclass/index.html

Reply:
I know that.  This is a fix for the old, stable xpaint.  In xpaint 3 I used eautoreconf, but I did not want to make big changes to a stable ebuild.  That's why I also added a comment.

If this is not acceptable I can add a proper fix, hoping I won't break anything.

Reply:
Please disregard the previous comment, I think I understand the problem now.  I'll come back with a new push later.

Reply:
I misunderstood last time, but now I hope it's right.  Please review.